src/Schema/Blueprint.php

```diff
@@ -276,6 +277,14 @@ public function drop()
         $this->collection->drop();
     }
 
+    /** @inheritdoc */
+    public function renameColumn($from, $to)
+    {
+        $this->collection->updateMany([$from => ['$exists' => true]], ['$rename' => [$from => $to]]);
+
+        return $this;
+    }
+
```

Comment:
First, it finds the documents that have the `$from` column and then runs a query to rename their column to the `$to`.